go/apirouter/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"github.com/cnotch/apirouter"
+	"net/http"
+	"io"
+)
+
+
+func main() {
```

Comment:
@cnotch @System-Glitch I've changed the main method. As I understand this way is considered as "standard usage" in `go` 
https://pkg.go.dev/net/http#example-ResponseWriter-Trailers

Feel free to correct me if I'm wrong

Reply:
PS : There is no need to output `json` (at least yet). The idea is to output `plain/text`

Reply:
Direct IO theoretically provides better performance when formatting is not required